wger/core/management/commands/extract-i18n.py

```diff
@@ -46,6 +46,7 @@ def cleanup_name(name: str) -> str:
         data = [i for i in ExerciseCategory.objects.all()] \
             + [i for i in Equipment.objects.all()] \
             + [i for i in Muscle.objects.all()] \
```

Comment:
we can remove this line :)

Reply:
Nevermind, could remove this by fixing the merge conflicts

Reply:
Should I regen the translation files I saw `wger/locale/el/LC_MESSAGES/django.mo` get changed when running the commands I found on https://wger.readthedocs.io/en/latest/i18n.html

Reply:
sure, go ahead